### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by KaskMIL**
# Add index, style, liters, .gitignore and readme
## HTML
Add a heading
## Style
Add color
## Liters
Add Github Actions to master best practices
## .gitignore
Add a file that specifies intentionally untracked files to ignore
## Readme
Add a description of the proyect

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
```
Comment (m05e5) [inline index.html]: - [ ] Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @KaskMIL , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️

## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve
  
  - [x] Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO
  
  - [x] Your Pull Request should have a descriptive title telling you what changes are inside.
  
  - [x] Your Pull Request should have a summary describing introduced changes in more detail (you can edit it at the top of the PR)

_I'm sure you can do it just keep it up_
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (KaskMIL) [reply index.html]: Hi @m05e5,
Thanks for your review, it helps me a lot.

Comment (m05e5) [reply index.html]: You're welcome :) I'm glad I could help and I'm sure you'll become a great developer. Keep it up

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**